test/functional/test_framework/test_framework.py

```diff
@@ -187,7 +183,7 @@ def main(self):
                 for fn in filenames:
                     try:
                         with open(fn, 'r') as f:
-                            print("From" , fn, ":")
+                            print("From", fn, ":")
```

Comment:
Not sure how to feel about the whitespace changes. This might discourage or distract reviewers. Maybe we could make those in a separate somewhat larger pull?